acceptance-tests/src/test/java/it/com/atlassian/bitbucket/jenkins/internal/SmokeTest.java

```diff
@@ -193,7 +193,7 @@ public void testRunFolderCredentialsMultibranch() throws Exception {
     @Test
     @Ignore("https://github.com/jenkinsci/acceptance-test-harness/issues/721")
     // This issue is fixed in the v1.109 of acceptance-test-harness but that version is not compatible with current
-    // Jenkins version (v2.289.1) we are using or the LTS (v2.319.1). Jenkins version should be >= v2.323 for v1.109 of
+    // Jenkins version (v2.319) we are using or the LTS (v2.319.1). Jenkins version should be >= v2.323 for v1.109 of
```

Comment:
I think we should merge this first, but now our minimum version is high enough we can address this smoke test failure :smile: